crates/net/network/src/manager.rs

```diff
@@ -147,6 +149,19 @@ impl<C> NetworkManager<C> {
     pub fn secret_key(&self) -> SecretKey {
         self.swarm.sessions().secret_key()
     }
+
+    #[inline]
+    fn update_poll_metrics(&self, start: Instant, poll_durations: NetworkManagerPollDurations) {
+        let metrics = &self.metrics;
+
+        let NetworkManagerPollDurations { acc_network_handle, acc_swarm } = poll_durations;
+
+        // update metrics for whole poll function
+        metrics.duration_poll_network_manager.set(start.elapsed());
+        // update poll metrics for nested items
+        metrics.duration_poll_network_handle.set(acc_network_handle.as_secs_f64());
+        metrics.duration_poll_swarm.set(acc_swarm.as_secs_f64());
```

Comment:
this should be µs

Reply:
I recommended to use `as_secs_f64` on another metrics PR because it captures us, and lower, is there any particular reason for using us instead?

Reply:
I see, I overlooked the f64 part here